django_cas_ng/views.py

```diff
@@ -109,8 +109,9 @@ def login(request, next_page=None, required=False):
         elif settings.CAS_RETRY_LOGIN or required:
             return HttpResponseRedirect(_login_url(service))
         else:
-            error = "<h1>Forbidden</h1><p>Login failed.</p>"
-            return HttpResponseForbidden(error)
+            #error = "<h1>Forbidden</h1><p>Login failed.</p>"
+            #return HttpResponseForbidden(error)
+            return HttpResponseRedirect(next_page)
```

Comment:
Sorry, this one is maybe a dirty fix to fit our specific need.
this is with `CAS_CREATE_USER = False`
and this in url.py :`url(r'^cas/?$', 'django_cas_ng.views.login',{'next_page':'check'}),`
So we can check if valid user in CAS is also known for this app.